tests/unittests/test_transformation.py

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env python
-# -*- coding: utf-8 -*-
-"""
-Copyright (c) 2018-present mundialis GmbH & Co. KG
+"""Copyright (c) 2018-present mundialis GmbH & Co. KG.
```

Comment:
Maybe change to 2018-2025?
```suggestion
"""Copyright (c) 2018-2025 mundialis GmbH & Co. KG.
```

Reply:
changed for all files